Cargo.toml

```diff
@@ -143,11 +143,13 @@ default-features = false
 "raft-proto:0.6.0-alpha" = { git = "https://github.com/pingcap/raft-rs", branch = "master" }
 "protobuf:2.8.0" = { git = "https://github.com/nrc/rust-protobuf", branch = "v2.8" }
 "protobuf-codegen:2.8.0" = { git = "https://github.com/nrc/rust-protobuf", branch = "v2.8" }
+"criterion:0.3.0" = { git = "https://github.com/YangKeao/criterion.rs.git", branch = "remove_outliers" }
```

Comment:
why not official criterion here?

Reply:
Official criterion doesn't remove outliers before calculate result. (and the author of criterion thinks we shouldn't simply remove outliers, so this feature will never appear in official criterion. )

Reply:
Now more functions are added in my repo. Such as quantile regression rather than simple least square regression which is still in discussing [here](https://github.com/bheisler/criterion.rs/issues/324)

Before I get the benchmark result more stable, the statistic way we used may change rapidly. Some of these features may be merged into official criterion in future while some of them will not. At least now I will continue to use my repo as an experiment.